pack/ur.json

```diff
@@ -110,6 +127,23 @@
         "type_code": "event",
         "uniqueness": false
     },
+    {
+        "code": "26075",
+        "cost": 2,
+        "deck_limit": 3,
+        "faction_code": "criminal",
+        "faction_cost": 2,
+        "flavor": "Return to sender.",
+        "illustrator": "Elizaveta Sokolova",
+        "pack_code": "ur",
+        "position": 75,
+        "quantity": 3,
+        "side_code": "runner",
+        "text": "When you install this hardware, choose an installed piece of ice. Use this hardware only during encounters with that ice.\n[trash]: Break up to 2 subroutines. When this run ends, if is successful, you may shuffle a copy of Boomerang from your heap into your stack.",
```

Comment:
"if is successful" -> "if it is successful"

Reply:
Thank you!